docs/content/latest/explore/backup-restore/point-in-time-recovery-ycql.md

```diff
@@ -267,7 +249,7 @@ Relative times can be in any of the following formats (again, note that you can
 1. At a terminal prompt, restore the snapshot you created earlier:
 
     ```sh
-    $ bin/yb-admin restore_snapshot_schedule 0e4ceb83-fe3d-43da-83c3-013a8ef592ca minus "5m"
+    $ bin/yb-admin restore_snapshot_schedule 0e4ceb83-fe3d-43da-83c3-013a8ef592ca minus 5m
```

Comment:
is this still worth keeping as a standalone example? I'm thinking maybe just link to the yb-admin section saying you can also do it with relative time

the reason I'd like to not have this in explore, is because of the "oh be sure to wait N minutes before using this flavor"

Reply:
I think it is, and i've adjusted the wording a bit from "wait five minutes" to "note the time..."